packages/compass-home/src/stores/store.js

```diff
@@ -42,15 +42,6 @@ store.onActivated = (appRegistry) => {
 
     store.dispatch(changeConnectionTitle(getConnectionTitle(connectionInfo) || ''));
 
-    const StatusAction = appRegistry.getAction('Status.Actions');
-    if (StatusAction) {
-      StatusAction.configure({
-        animation: true,
-        message: 'Loading navigation',
-        visible: true
-      });
-    }
```

Comment:
This "Loading navigation" overlay has a race with "Loading databases" one and I am not sure if this was intended or not, but it seemed to me that we can just remove it